### PR TITLE
Switch from Coveralls to Codecov code coverage service

### PR DESCRIPTION
**Pull request by tgerring**

### CODE REVIEW

**Comment by robotally** [comment]
| Vote | Count | Reviewers |
| :---: | :---: | :---: |
| :+1: | 0 |  |
| :-1: | 0 |  |

_Updated: Tue Aug 18 21:09:19 UTC 2015_

File: .travis.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ before_script:
 script:
   - make travis-test-with-coverage
 after_success:
-  - if [ "$COVERALLS_TOKEN" ]; then goveralls -coverprofile=profile.cov -service=travis-ci -repotoken $COVERALLS_TOKEN; fi
+  - bash <(curl -s https://codecov.io/bash)
```
Comment (Gustav-Simonsson) [inline .travis.yml]: hmm, not too optimal for security, can we put this file in the repo instead?

Comment (tgerring) [inline .travis.yml]: This is in the Travis script only, so the target vector is quite small. Committing the specific script version may cause random breaks if their backend becomes de-sync'd.

Comment (obscuren) [inline .travis.yml]: Acceptable

**Pull request merged**